manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -67,6 +72,11 @@ public Bridge getBridge(String id, String customerId) {
 
     @Override
     public void deleteBridge(String id, String customerId) {
+        List<Processor> processors = processorService.getProcessors(id, customerId);
+        if (processors.size() > 0) {
+            throw new BridgeLifecycleException("It is not possible to delete a Bridge instance with active Processors.");
```

Comment:
I think this is true only until we implement a delete cascade/delete all logic right? If so, do we have a ticket to mention here in a comment?

Reply:
https://issues.redhat.com/browse/MGDOBR-43